e4s_cl/cli/commands/init.py

```diff
@@ -441,6 +442,15 @@ def main(self, argv):
         selected_profile = Profile.selected()
         requested_name = getattr(args, 'profile_name', None)
 
+        # Determine if wi4mpi is needed depending on mpi version detected
+        installed, vendor = check_wi4mpi(selected_profile)
```

Comment:
We should extract the MPI library path at this step. It is required to:
- Check the vendor version and determine if it requires Wi4MPI;
- Output the vendor and version to set the profile name;
- Modify the `wi4mpi.cfg` configuration file to point the installation towards the MPI library.

Instead of looking for the path in the list multiple time, getting this information here could simplify things.